### PR TITLE
Updated SChinese Announcement

### PR DESCRIPTION
**Pull request by CrewCyan**
Update Simplified Chinese Announcement(Version numbers: 1.1.0 and 1.1.1)

### CODE REVIEW

**Comment by NikoCat233** [comment]
这个翻译挺生硬啊...

**Comment by CrewCyan** [comment]
> 这个翻译挺生硬啊... 而且有的地方反了错了

你干嘛嗨嗨呦（bushi）

**Review by Tommy-XL — APPROVED** [review]
It's weird that you put this below version 1.0.1, but otherwise everything is ok, lol.

**Comment by NikoCat233** [comment]
> It's weird that you put this below version 1.0.1, but otherwise everything is ok, lol.

This translation is machine based and contains mutiple errors.
Pls wait while i change some sentences

**Comment by NikoCat233** [comment]
> > This translation is machine based and contains mutiple errors. Pls wait while i change some sentences
> 
> Then I'll close this pull request later

照着review改一下就可以了

File: Patches/AnnouncementPatch.cs [hunk 1]
```diff
@@ -406,11 +406,11 @@ public static void Init()
                 //TOHE v1.0.1
                 var news = new ModNews
                 {
-                    Number = 80004,
+                    Number = 80003,
                     Title = "Town of Host: Enhanced v1.0.1",
                     SubTitle = "★★ 新时代 ★★",
                     ShortTitle = "TOH: Enhanced v1.0.1",
-                    BeforeNumber = 80003,
+                    BeforeNumber = 80002,
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:413]: 这里的公告编号有点错误

File: Patches/AnnouncementPatch.cs [hunk 2]
```diff
@@ -424,6 +424,86 @@ public static void Init()
...
+                    ShortTitle = "TOH: Enhanced v1.1.1",
+                    BeforeNumber =80004,
+                    Text = "<size=150%>欢迎来到TOH: Enhanced v1.1.1!</size>\n" +
+                    "\n<b>【基于】</b>\n - 基于TOH: Enhanced v1.1.0\r\n" +
+
+                    "\n<b>【Bug修复】</b>" +
+                        "\n     - 修正了（坏）迷你船员不可审判或复仇的错误" +
+                        "\n     - 修复了换票师和其中的几个错误,保持实验性" +
+                        "\n     - 修正了狂战士无法死亡和垃圾信息的错误" +
+                        "\n     - 修复了疯狂的好迷你船员的问题" +
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:441]: 这里应该是背叛的好迷你船员，而不是疯子

File: Patches/AnnouncementPatch.cs [hunk 3]
```diff
@@ -424,6 +424,86 @@ public static void Init()
...
+                        "\n     - 修正了一个与决胜局和无效投票的冲突\n\r" +
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:442]: 这里是破平者，而不是决胜局

File: Patches/AnnouncementPatch.cs [hunk 4]
```diff
@@ -424,6 +424,86 @@ public static void Init()
...
+                    BeforeNumber = 80003,
+                    Text = "<size=150%>欢迎来到TOH: Enhanced v1.1.0!</size>\n" +
+                    "\n<b>【基于】</b>\n - 基于TOH: Enhanced v1.0.1\r\n" +
+
+                    "\n<b>【新职业/附加职业】</b>" +
+                        "\n     - 总统(船员:权力)" +
+                        "\n     - 间谍(船员:支援)" +
+                        "\n     - 义务警长(船员:击杀)\n\r" +
+
+                        "\n     - 反弹(附加职业:混合)\n\r" +
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:470]: 这里是回弹者

File: Patches/AnnouncementPatch.cs [hunk 5]
```diff
@@ -424,6 +424,86 @@ public static void Init()
...
+
+                    "\n<b>【新设置】</b>" +
+                        "\n     - 船员哨兵:每完成x(用字母x表示设定的任务数)项任务后，立即扑杀并击杀" +
+                        "\n     - 死亡契约:死亡契约中的玩家可以召开紧急会议" +
+                        "\n     - 龙卷风:隐藏玩家的交换对象\n" +
+
+                        "\n     - 欺骗者:失去使用错误欺骗的能力" +
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:477]: 这里是赝品商

File: Patches/AnnouncementPatch.cs [hunk 6]
```diff
@@ -424,6 +424,86 @@ public static void Init()
...
+                        "\n     - 商人:只能销售已启用的附加职业" +
+                        "\n     - 验尸官:告知凶手已被追踪\n\r" +
+
+                        "\n     - 感染者:双击即可杀死/感染" +
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:481]: 不应该说杀死，而是击杀

File: Patches/AnnouncementPatch.cs [hunk 7]
```diff
@@ -424,6 +424,86 @@ public static void Init()
...
+                        
+                        "\n     - 迷幻者:杀手可获得迷惑的视觉\n\r" +
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:483]: 应该是：带刀玩家可以获得迷幻者的视野

File: Patches/AnnouncementPatch.cs [hunk 8]
```diff
@@ -424,6 +424,86 @@ public static void Init()
...
+                        "\n     - 附加职业:光辉" +
+
+                    "\n<b>【其他变化】</b>" +
+                        "\n     - 支持新语言:葡萄牙语" +
+                        "\n     - 新大厅游戏：/rps&/抛硬币" +
+                        "\n     - 将特工更名为邪恶追踪者" +
+                        "\n     - 将破坏者重新命名为反对管理员" +
+                        "\n     - 新迷彩皮肤" +
+                        "\n     - 添加了Default_Template.txt" +
+                        "\n     - 返场遐想者和仇恨者" +
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:497]: 不是仇恨者，是FFF团

**Review by LezaiYa — COMMENTED** [review]
中文翻译的错误指正

**Comment by Tommy-XL** [comment]
Is done?

**Comment by Tommy-XL** [comment]
Looks like yes, lol

**Pull request merged**